v2/lua/openresty_org/controller.lua

```diff
@@ -55,6 +68,29 @@ function _M.run()
         return ngx.redirect(uri .. "/", 301)
     end
 
+    local from = re_find(uri, [[ ^ /yum/ ( cn/ )? ( [a-z-0-9]+ ) /OpenResty.repo $ ]], 'jox')
+    if from then
+        -- Skip /yum/
+        from = from + 5
+        resp_header["Content-Type"] = "text/plain"
+        if sub(uri, from, from + 2) == 'cn/' then
```

Comment:
I personally prefer `re_match`, `m[1]`, `m[2]` instead of this, it will make the code more simpler and bug less :)